test/unit/runtime/Loader.js

```diff
@@ -108,9 +111,9 @@ suite('Loader.js', function() {
 
   test('LoaderModule', function(done) {
     var code =
-        'import * as a from "./resources/test_a.js";\n' +
-        'import * as b from "./resources/test_b.js";\n' +
-        'import * as c from "./resources/test_c.js";\n' +
+        'import * as a from "./test/unit/runtime/resources/test_a.js";\n' +
```

Comment:
Per our earlier discussion with @guybedford we now use
```
function getTestLoader() {
  return new System.constructor();
}
```
The baseurl for this loader will be the node working directory.